infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java

```diff
@@ -30,6 +31,15 @@
 
   public static final DeserializableTypeDefinition<Byte> BYTE_TYPE = new ByteTypeDefinition();
 
+  public static final DeserializableTypeDefinition<Bytes> BYTES_TYPE =
+      DeserializableTypeDefinition.string(Bytes.class)
+          .formatter(Bytes::toHexString)
+          .parser(Bytes::fromHexString)
+          .example("0x01")
+          .description("Bytes hexadecimal")
+          .format("byte")
+          .build();
```

Comment:
we'll try to be more specific than this in typing if possible. I think we've used mostly for signatures above which will hopefully disappear.
The downside of having this is it's unbounded, so it could be used for bad things...